test/e2e/fixtures/texts.json

```diff
@@ -15,6 +15,7 @@
   "prev": "Forrige",
   "next": "Neste",
   "requiredField": "Feltet er påkrevd",
+  "requiredFieldFromBackend": "Feltet er påkrevd (fra backend)",
```

Comment:
Oooh, sorry about that one! :see_no_evil: I might have changed that label without thinking all that much, but thanks for making the change here. I guess a new deployment of `frontend-test` is needed as well.

I've heard people ask why the older error message stayed just for this field here, that's why I added the note about this coming from backend.

Reply:
No worries! 😄 I already deployed a new version of frontend-test which is how i noticed this 😅 I think frontend-test should be good if nothing else pops up.